scripts/speech_recognition/code_switching/code_switching_manifest_creation.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import logging
+import os
+import random
+
+# Checks -
+# Please normalize the text for each language so that we have only the alphabets (no numbers, special characters, punctuation)
```

Comment:
Should these be checks in the source?

Reply:
This is a actually a recommendation and not a necessary condition. The script would not fail or work incorrectly if we have numbers/punctuations/special characters, but to train better ASR models it would be beneficial to have the text normalized first

Reply:
fixed